src/test/java/org/eclipse/microprofile/starter/core/TemplateVariableProviderTest.java

```diff
@@ -37,6 +39,8 @@ public void shouldStoreValidApplicationName() {
         JessieMaven maven = new JessieMaven();
         JessieSpecification specification = new JessieSpecification();
         specification.setMicroProfileVersion(MicroProfileVersion.MP22);
+        specification.setBuildTool(BuildTool.MAVEN);
```

Comment:
There is no gradle test. right?

Reply:
This is not a complete test with Gradle (or Maven).  Just about creating a default for Application Name.

But added a test for Gradle.